utils/convert-month-partitioned-parts/main.cpp

```diff
@@ -97,6 +97,7 @@ void run(String part_path, String date_column, String dest_path)
     Poco::File(new_tmp_part_path_str + "checksums.txt").setWriteable();
     WriteBufferFromFile checksums_out(new_tmp_part_path_str + "checksums.txt", 4096);
     checksums.write(checksums_out);
+    checksums.close();
```

Comment:
@azat 

`MergeTreeDataPartChecksums` doesn't define `close` method, do you mean `checksums_in.close()`

Reply:
Indeed, my bad, can you send a patch? (Interesting, there is no splitted check on CI, hence this wasn't captured, since ENABLE_UTILS is set only for splitted)

Reply:
@baibaichen @azat #20513